lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -444,8 +447,36 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain(self):
```

Comment:
You may use `search_by_name()' method:

```
search_by_name(
    self._connection.system_service().storage_domains_service(),
    self._module.params['storage_domain']
)
```

https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/ovirt.py#L221